egs/voxpopuli/ASR/local/separate_punctuation.py

```diff
@@ -0,0 +1,131 @@
+#!/bin/env python3
+# Copyright    2023  Brno University of Technology (authors: Karel Veselý)
+#
+# See ../../../../LICENSE for clarification regarding multiple authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This script chops the punctuation as standalone tokens.
+Example:
+    input: "This is fine. Yes, you are right."
+    output: "This is fine . Yes , you are right ."
+
+The script also handles exceptions in a hard-coded fasion.
+
+(same functionality could be done with `nltk.tokenize.word_tokenize()`,
+ but that would be an extra dependency)
+
+It can be used as a module, or as an executable script.
+
+Usage example #1:
+  from separate_punctuation import separate_punctuation
+
+Usage example #2:
+  python3 ./local/separate_punctuation.py \
+    --ignore-columnts 1 \
+    ${kaldi_data}/text
```

Comment:
This code seems to read the content from `stdin`, not the provided file `${kaldi_data}/text`. Is it correct?